webapp/redux/middleware/plotMiddleware.js

```diff
@@ -98,31 +101,19 @@ const plotFigure = async (plotId, plotMethod, plotType = false, theme) => {
     }
     if (plotMethod.startsWith("iplot")) {
       let html_text = response.replace ? response.replace(/\\n/g, '').replace(/\\/g, '') : ''
-
       if (plotId === "rxdConcentrationPlot") {
         // FIXME: How can we center the bokeh plots when sizing_mode='scale_height'
         html_text = html_text.replace("<head>", "<head><style>.bk {margin: 0 auto!important;}</style>")
       }
-      setPlotToWindow(plotId, html_text)
+      return html_text;
     } else if (response?.length !== undefined) {
-      setPlotToWindow(plotId, response[0]);
-    } else if (response == -1) {
+      return response[0];
+    } else if (response === -1) {
       return null;
     } else {
-      setPlotToWindow(plotId, response);
+      return response;
     }
-    return response;
   } catch (error) {
     console.error(error);
   }
-  
-
-}
-
-const setPlotToWindow = (plotId, svgResponse) => {
-  if (svgResponse === '') {
-    console.log("No plot to show")
-    return
-  }
-  window.plotSvgImages[plotId] = svgResponse
```

Comment:
It's html, not svg

Reply:
It was both, the name is just misleading, see how it was used in WidgetFactory.ts.